llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp

```diff
@@ -459,16 +491,20 @@ struct LinalgStrategyTilePass
     MLIRContext *ctx = funcOp.getContext();
     RewritePatternSet tilingPattern(ctx);
     if (!anchorOpName.empty())
-      tilingPattern.add<LinalgTilingPattern>(anchorOpName, ctx, options,
-                                             filter);
+      tilingPattern.add<LinalgSCFTilingPattern>(anchorOpName, ctx, options,
+                                                filter);
     else
-      tilingPattern.add<LinalgTilingPattern>(ctx, options, filter);
-    if (anchorOpName == tensor::PadOp::getOperationName())
-      populatePadTensorTilingPatterns(tilingPattern, options);
+      tilingPattern.add<LinalgSCFTilingPattern>(ctx, options, filter);
+    if (anchorOpName == tensor::PadOp::getOperationName()) {
```

Comment:
actually the upstream tiling handles pad ops as well. So you should be able to drop line 498 - 502.

Reply:
Done.